app/src/ui/clone-repository/clone-github-repository.tsx

```diff
@@ -95,4 +100,10 @@ export class CloneGithubRepository extends React.PureComponent<
       </DialogContent>
     )
   }
+
+  private onItemClicked = (repository: IAPIRepository, source: ClickSource) => {
+    if (source.kind === 'keyboard' && source.event.key === 'Enter') {
```

Comment:
We could pull this logic out into `cloneIfCloningEnabled` in the parent component. That way, the parent is in control over what key triggers the event.